src/tools/rbd_mirror/PoolWatcher.cc

```diff
@@ -72,7 +74,7 @@ void PoolWatcher::refresh_images(bool reschedule)
   // about new/removed mirrored images
 }
 
-void PoolWatcher::refresh(ImageIds *image_ids) {
+int PoolWatcher::refresh(ImageIds *image_ids) {
```

Comment:
@dillaman Just wander, below we are ignoring list_images_v2 failure. May be it is safer just to return error there too?